microservices/Order.py

```diff
@@ -0,0 +1,132 @@
+            street = request.form['street'].strip(),
+            city = request.form['city'].strip(),
+            state = request.form['state'].strip(),
+            zipCode = request.form['zipCode'].strip(),
+            phone = request.form['phone'].strip(),
+            email = request.form['email'].strip(),
+            website = request.form['website'].strip(),
+            cuisine = request.form['cuisine'].strip(),
+            description = request.form['description'].strip(),
+            delivery = request.form['delivery'].strip(),
+            user_id = login_session['user_id']
+        )
+        # add neworder item to db stage
+        session.add(neworder)        # commit neworder item to db
+        session.commit()
+
+        # flash message to indicate success
+        flash("New order: " + neworder.name + " ==> Created!")
+        # redirect user to updated list of orders
+        return redirect(url_for('showorders'))
+    else:
+        return render_template('neworder.html', title = "New order Input")
+
+# Edit a order
+#########################
+@app.route('/order/order id/edit/', methods = ['GET', 'POST'])
+def editorder(order_id):
+
+    # query db by order_id and assign to order variable
+    order = session.query(order).filter_by(id = order_id).one()
+
+    # Check to see if a user is logged in.
+    if 'username' not in login_session:
+        return redirect('/login')
+
+    # Check if user_id matches the user_id stored in login_session
+    if order.user_id != login_session['user_id']:
+        return authorizationAlert("Edit")
+
+    # If user is logged in then access editorder page
+    if request.method == 'POST':
+        if request.form:
+            # Store edited field data and POST to db
+            order.name = request.form['name'].strip()
+            order.street = request.form['street'].strip()
+            order.city = request.form['city'].strip()
+            order.state = request.form['state'].strip()
+            order.zipCode = request.form['zipCode'].strip()
+            order.phone = request.form['phone'].strip()
+            order.email = request.form['email'].strip()
+            order.website = request.form['website'].strip()
+            order.cuisine = request.form['cuisine'].strip()
+            order.description = request.form['description'].strip()
+            order.delivery = request.form['delivery'].strip()
+        # add editorder data to db stage
+        session.add(order)
+        # commit editorder data to db
+        session.commit()
+
+        # flash msg to indicate success
+        flash("Edited order: " + order.name + " ==> Updated!")
+        # redirect user to updated list of orders
+        return redirect(url_for('showorders'))
+    else:
+        # Render the html needed to edit the order.
+        return render_template('editorder.html', title = 'Edit order', order = order)
+
+# Delete an order
+
+@app.route('/order/order id/delete/', methods = ['GET', 'POST'])

```

Comment:
you should be able to just make this a DELETE request... if not, a POST should suffice